### PR TITLE
tests: fix incompatibility with pytest>=2.8.0

### PR DESCRIPTION
**Pull request by SamiHiltunen**
* FIX Removes calls to PluginManager
  consider_setuptools_entrypoints() removed in PyTest 2.8.0.

Signed-off-by: Sami Hiltunen <sami.mikael.hiltunen@cern.ch>

### CODE REVIEW

**Comment by jirikuncar** [comment]
- [ ] fix wrong `invenio_testsuite` import
- [ ] bump `invenio-base` dependency

**Comment by SamiHiltunen** [comment]
@jirikuncar points have been addressed

File: tests/test_record.py [hunk 1]
```diff
@@ -22,24 +22,33 @@
...
 class TestRecord(InvenioTestCase):
+
     """Record - demo file parsing test."""
 
     def test_records_created(self):
         """Record - demo file how many records are created."""
         xmltext = pkg_resources.resource_string(
-            'invenio.testsuite',
-            os.path.join('data', 'demo_record_marc_data.xml'))
+            'invenio_records',
```
Comment (jirikuncar) [inline tests/test_record.py]: this is not correct location - better use `os.path.join(dirname(__file__), 'data', 'demo_....xml')`

File: tests/test_record.py [hunk 2]
```diff
@@ -472,12 +483,15 @@ def test_rec_json_creation_from_marcxml(self):
 
         assert 'main_entry_personal_name' in r
         assert 'added_entry_personal_name' in r
-        assert r['main_entry_personal_name']['personal_name'] == "Efstathiou, G P"
+        assert r['main_entry_personal_name'][
```
Comment (jirikuncar) [inline tests/test_record.py]: please do not re-format assert statements

**Pull request merged**